garden/package.json

```diff
@@ -28,6 +29,7 @@
     "start": "react-scripts start & npx tailwindcss -i ./src/App.css -o public/main.css --watch",
     "start-https": "HTTPS=true react-scripts start & npx tailwindcss -i ./src/App.css -o public/main.css --watch",
     "build": "react-scripts build && npx tailwindcss -i ./src/App.css -o public/main.css",
+    "remove-mock-service-worker": "sh -c \"rm -f build/mockServiceWorker.js\"",
```

Comment:
TODO: confirm on a staging build that we're not serving the dev-only msw js file

Reply:
I'll inspect this after merging and adjust if needed